content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,238 @@
+
+
+## 3 - Create Teams in Palette
+
+The remaining step is to create teams in Palette for the group claims that you configured in AD FS, and give them the appropriate permissions. For this example, you will create the `Admins` team and give it **Tenant Admin** permissions. You can repeat this for any other team that you have configured group claims.
+
+25. Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **Users & Teams** --> **Teams** tab. Click on the **+ Create Team** button.
+
+![Create Palette Team](palette-sso-with-adfs-images/21-create_team.png)
+
+
+26. Specify `Admins` in the **Team name** field. You don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
+
+![Name Palette Team](palette-sso-with-adfs-images/22-name_team.png)
+
+
+27. You will be returned back to the list of teams. Click on the newly created **Admins** team to review the details page. To give this team administrative access to the entire tenant and all the projects in it, assign the **Tenant Admin** role. Click on **Tenant Roles** --> **+ Add Tenant Role**:
+
+![Palette Tenant Roles](palette-sso-with-adfs-images/23-tenant_roles.png)
+
+
+28. Click on **Tenant Admin** to enable the role. Click **Confirm** to add the role.
+
+![Add Tenant Role](palette-sso-with-adfs-images/24-add_tenant_role.png)
+
+
+You will receive a message stating **Roles have been updated**. Repeat this procedure for any other teams but ensure they are given the appropriate permissions. 
+
+29. Click the **X** next to **Team Details** in the top left corner to exit this screen.
+
+You have now successfully configured Palette SSO based on OIDC with Microsoft AD FS!
+
+
+# Validation
+30. Log in to Palette through SSO as a user that is a member of the `SpectroTeam - Admins` group in Active Directory to verify that users are automatically added to the `Admins` group in Palette.
+
+If you're still logged into Palette with a non-SSO user, log out by clicking on the **User Menu** at the top right and selecting **Logout**.
+
+![User Logout](palette-sso-with-adfs-images/25-user_logout.png)
+
+
+31. The Palette login screen now displays a **Sign in** button and no longer presents a username and password field. Below the **Sign In** button, there is an **SSO issues? --> Use your password** link. This link can be used to bypass SSO and log in with a local Palette account in case there is an issue with SSO and you need to access Palette without SSO.
+
+Click on the **Sign in** button to log in via SSO.
+
+![User SSO Login](palette-sso-with-adfs-images/26-palette_login.png)
+
+
+32. If this is the first time you are logging in with SSO, you will be redirected to the Microsoft AD FS login page. Depending on your organization's SSO settings, this could be a simple login form or require MFA (Multi-Factor Authentication).
```

Comment:
```suggestion
3. If this is the first time you are logging in with SSO, you will be redirected to the Microsoft AD FS login page. Depending on your organization's SSO settings, this could be a simple login form or require MFA (Multi-Factor Authentication).
```